pennylane/templates/subroutines/trotter.py

```diff
@@ -126,7 +126,20 @@ def my_circ():
     .. details::
         :title: Usage Details
 
-        One can recover the behaviour of :class:`~.ApproxTimeEvolution` by setting :code:`order=1`.
+        Note, this operation is similarily to the :class:`~.ApproxTimeEvolution`. Identical behaviour can 
+        be replicated by flipping the order of terms in the input hamiltonian, setting :code:`order=1`, 
+        and setting :code:`time = -1 * time`.
+
+        >>> time, n = (0.25, 1)
+        >>> coeffs, ops = ([1, 2], [qml.PauliX(0), qml.PauliZ(0)])
+        >>> h, h_flipped = (qml.Hamiltonian(coeffs, ops), qml.Hamiltonian(coeffs[::-1], ops[::-1]))
+        >>> 
+        >>> m_trotter_p = qml.matrix(qml.TrotterProduct(h, time, order=1, n=n))
+        >>> m_approx_te = qml.matrix(qml.ApproxTimeEvolution(h_flipped, -1*time, n=n))
+        >>> np.allclose(m_trotter_p == m_approx_te)
+        True
+
```

Comment:
This feels too much like a test. Can we keep it simple with:

```suggestion
        and setting :code:`time = -1 * time`. The order of terms in a Hamiltonian
        ``h`` can be reversed using

        >>> h = qml.Hamiltonian(h.coeffs[::-1], h.ops[::-1])
```

Reply:
Sure! I just wanted to make sure it was thorough enough that a user would know exactly how to convert. This should be sufficient.